pkg/controller/import-controller.go

```diff
@@ -638,7 +641,9 @@ func (r *ImportReconciler) requiresScratchSpace(pvc *corev1.PersistentVolumeClai
 		case SourceGlance:
 			scratchRequired = true
 		case SourceRegistry:
-			scratchRequired = true
+			if pvc.Annotations[AnnRegistryImportMethod] != "cri" {
```

Comment:
can you make "cri" a constant, it is used in a lot of places.

Reply:
Fixed. scratchRequired is sure false in this method, right?

Reply:
Fixed

Reply:
you can write this as:
```go
  scratchRequired = pvc.Annotations[AnnRegistryImportMethod] != "cri"
```
With the "cri" constant of course.

Reply:
Right, fixed.